custom_components/bosch_alarm/__init__.py

```diff
@@ -8,6 +8,7 @@
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr
```

Comment:
i would drop the "as dr" here, and write
`dr = device_registry.async_get(hass)`
below

Reply:
Fair enough, was just copying what was being done in home assistant itself